src/View.zig

```diff
@@ -0,0 +1,161 @@
+//! A View is effectively an "oversized" Window that can be written to and rendered in pieces.
+
+const std = @import("std");
+const mem = std.mem;
+
+const View = @This();
+
+const Screen = @import("Screen.zig");
+const Window = @import("Window.zig");
+const Unicode = @import("Unicode.zig");
+const Cell = @import("Cell.zig");
+
+/// View Allocator
+alloc: mem.Allocator,
+/// Underlying Screen
+screen: *Screen,
+/// Underlying Window
+win: *Window,
```

Comment:
We don't need to store a window at all here. They should be created on-the-fly.

Reply:
This goes back to API simplicity. View will always need a Window for its function calls, but it also only uses a subset of the Window API. This allows View to control its API without having to reinvent the wheel with the great functions available from Window. 

The other option is to store the View's `width` and `height` directly to the View and create the Window during each function call. This would further remove the need for Screen allocation, but it would open up Views to having their size changed on the fly (which might not be desirable).